homeassistant/data_entry_flow.py

```diff
@@ -454,7 +454,7 @@ def show_advanced_options(self) -> bool:
         return self.context.get("show_advanced_options", False)
 
     def add_suggested_values_to_schema(
-        self, data_schema: vol.Schema, suggested_values: Mapping[str, Any]
+        self, data_schema: vol.Schema, suggested_values: Mapping[str, Any] | None
```

Comment:
I think it makes sense to add `None` support here, as user_input in config flows is usually optional.
It removes the need for a check in every integration.

Without this, then we need to do either:
```python
data_schema = self.add_suggested_values_to_schema(DATA_SCHEMA, user_input or {})
```
or
```python
data_schema = DATA_SCHEMA
if user_input:
    data_schema = self.add_suggested_values_to_schema(DATA_SCHEMA, user_input)
```